docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "2.1"
+
+services:
+  sonarqube:
+    depends_on:
+      - db
+    image: sonarqube-with-community-branch-plugin:7.9-community
+    build:
+      context: .
```

Comment:
I've added the build key, so that you can do

    docker-compose build
    docker-compose up

to easily test any local changes of the plugin.